azerothcore/azerothcore-wotlk
# Corrected MovementType values for three Dustbelcher NPCs

## Changes Proposed:
- Modified the MovementType values for three Dustbelcher NPCs to resolve WaypointMovementGenerator::LoadPath: creature doesn't have waypoint path id error.
- Specifically, the errors generated in the worldserver console were: 
`WaypointMovementGenerator::LoadPath: creature Dustbelcher Mauler (GUID Full: 0xf130000a9d0016b4 Type: Creature Entry: 2717  Low: 5812) doesn't have waypoint path id: 0 WaypointMovementGenerator::LoadPath: creature Dustbelcher Wyrmhunter (GUID Full: 0xf130000a9c0016b7 Type: Creature Entry: 2716  Low: 5815) doesn't have waypoint path id: 0 WaypointMovementGenerator::LoadPath: creature Dustbelcher Shaman (GUID Full: 0xf130000a9e0016b8 Type: Creature Entry: 2718  Low: 5816) doesn't have waypoint path id: 0 WaypointMovementGenerator::LoadPath: creature Dustbelcher Wyrmhunter (GUID Full: 0xf130000a9c0016b9 Type: Creature Entry: 2716  Low: 5817) doesn't have waypoint path id: 0 WaypointMovementGenerator::LoadPath: creature Dustbelcher Mystic (GUID Full: 0xf130000b5b0016ba Type: Creature Entry: 2907  Low: 5818) doesn't have waypoint path id: 0`
- These were due to NPCs having the wrong MovementType set as these NPCs did not have paths supporting MovementType=2.

## Issues Addressed:

- Closes #16485 

## SOURCE:


## Tests Performed:

- Yes. SQL changes only so build is not affected. Tested to see that NPCs in-game were adhering to new MovementType.

## How to Test the Changes:


1. Spawn NPC GUID 6974, 6989, or 7210.
2. Observe movement.
3. Check worldserver console/logs to see that error is no longer present.

## Known Issues and TODO List:


- [ None ]


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
